src/pie-chart/labels.tsx

```diff
@@ -45,7 +45,6 @@ function LabelElement({
 }: LabelElementProps) {
   return (
     // Reset the transform property to prepare for `balanceLabelNodes`.
-    // The dataset attributes are also needed in the function for IE11 support.
```

Comment:
Didn’t find any references how these data attributes are related to IE11 support